### ISSUE TITLE
protobuf-dt does not recognize java_package and other options

### ISSUE THREAD

**Issue by jfancher** [google/protobuf-dt#4]
Similar to #2, but this is happening for options that do appear in the standard `descriptor.proto`

Upon upgrading to 2.3.1 (I think I was previously using 2.2.1) I'm seeing this error for almost all file and field options, including `java_generate_equals_and_hash`, `java_package`, `java_outer_classname`, `packed`, `deprecated`, to name a few. `default` is the only one I've noticed that is still recognized correctly.

As in #2 the error printed in the Problems view is `Couldn't resolve reference to IndexedElement '(name)'`.

I tried pointing to a custom `descriptor.proto` that definitely includes the options just to be sure, and that did not change anything.

**Comment by atrookey** [comment]
jfancher,

Thank you for the message!  There was a bug introduced in the scoping of the file descriptor.proto in the latest version of the editor.  I am working on a fix now.

**Comment by atrookey** [comment]
I forgot to resolve the default descriptor path stored in preferences with the actual resource location of descriptor.proto.  The editor was unable to interpret "google/protobuf/descriptor.proto" as "platform:/plugin/com.google.eclipse.protobuf/descriptor.proto".

**Comment by atrookey** [comment]
Just submitted a patch, https://github.com/google/protobuf-dt/commit/dce2d238ec644febeb314ac4c829efd123dcca36.  I will push a binary to the update site shortly.

**Comment by atrookey** [comment]
Uploaded binary to update site, https://github.com/google/protobuf-dt/commit/3bf458ac7c7583b01277664e58b84075936c215d.  May you please verify that the issue is fixed?

**Comment by atrookey** [comment]
A new binary has been added to the update site.   May you please verify that the problem is resolved?

**Comment by jfancher** [comment]
Yes, everything seems to be working with the new version. Thanks!

**Comment by atrookey** [comment]
Glad to hear it!

**Comment by gfecher** [comment]
The descriptor.proto file deployed with 2.3.2.201609161849 seems to contain either an old or a custom version of the file - e.g. it does not contain an entry for 'csharp_namespace' and lots of others.
I'm using Eclipse Neon.3 (4.6.3).

I did not manage to set/override the Options definition file either (I tried with and without a 'file:///' prefix). Is this supposed to be a file within the project? It is also not clear when changing this option should take effect - instantly or on restart?

In the end I injected in an up-to-date version of the descriptor.proto file into the plugin jar under .p2\pool\plugins\com.google.eclipse.protobuf_2.3.2.201609161849.jar and restarted Eclipse which resolved this.
